src/main.cpp

```diff
@@ -5971,8 +5973,13 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
                             // but it is an out parameter of IsManifestAuthorized.
                             unsigned int banscore_out = 0;
 
-                            if (CScraperManifest::IsManifestAuthorized(manifest.pubkey, banscore_out))
+                            // Also don't send a manifest that is older than the manifest retention period.
+                            int64_t nTimeCutoff = GetAdjustedTime() - SCRAPER_CMANIFEST_RETENTION_TIME;
```

Comment:
Can we use the `CScraperManifest::IsManifestCurrent()` function you added last round? It accounts for the scraper sleep interval, so its not quite the same, but maybe it makes sense to avoid sending manifests that will expire next loop iteration anyway.

Reply:
Yep. I think that is better and I will make that change.